generator/src/main/resources/com/github/fommil/netlib/generator/netlib-java.stg

```diff
@@ -84,6 +84,8 @@ public abstract class $name$ {
           break;
         } catch (Throwable e) {
           log.warning("Failed to load implementation from: " + className);
+          log.warning("Error was: " + e.toString());
+          e.printStackTrace();
```

Comment:
nope. This is a terrible idea. You can't just do printing stacktraces on people. This uses the logging system so that users can choose to ignore it... you're intentionally filling their stderr with noise and most people don't care.